Google.Api.Gax.Grpc/Rest/PartialDecodingStreamReader.cs

```diff
@@ -92,9 +93,11 @@ internal class PartialDecodingStreamReader<TResponse> : IAsyncStreamReader<TResp
     public TResponse Current { get; private set; }
 
     /// <summary>
-    /// Creates the StreamReader
+    /// Creates a new instance which will read data from the TextReader provided by
+    /// <paramref name="textReaderTask"/>, and convert each object within a top-level JSON array
+    /// into a response using <paramref name="responseConverter"/>.
     /// </summary>
-    /// <param name="textReaderTask">A stream reader returning partial JSON chunks</param>
+    /// <param name="textReaderTask">A task to provide text reader returning partial JSON chunks</param>
     /// <param name="responseConverter">A function to transform a well-formed JSON object into the proto message.</param>
     public PartialDecodingStreamReader(Task<TextReader> textReaderTask, Func<string, TResponse> responseConverter)
```

Comment:
Instead of receiving the TextReader here, could we receive something else from which to build the TextReader from, in case that makes disposing easier?

Reply:
I don't think so - I think this is the right level of abstraction here, and disposing should be okay.